rotkehlchen/tests/unit/test_ethereum_airdrops.py

```diff
@@ -337,3 +420,45 @@ def test_airdrop_fail(database, data_dir, messages_aggregator):
             database=database,
             data_dir=data_dir,
         )
+
+
+@pytest.mark.parametrize('remote_etag', ['etag', 'updated_etag'])
+@pytest.mark.parametrize('database_etag', [None, 'etag', 'updated_etag'])
+def test_fetch_airdrops_metadata(database, remote_etag, database_etag):
+    if database_etag is not None:
+        # if database_etag is present, add those values in DB
+        with GlobalDBHandler().conn.write_ctx() as write_cursor:
+            globaldb_set_unique_cache_value(
+                write_cursor=write_cursor,
+                key_parts=(CacheType.AIRDROPS_HASH, 'Metadata'),
```

Comment:
you would use the constant literal for etag here and in other parts of the file.